u19_pipeline/recording.py

```diff
@@ -0,0 +1,181 @@
+import datajoint as dj
+from u19_pipeline import lab, task, subject
+
+schema = dj.schema('u19_recording')
+
+from u19_pipeline import acquisition, subject
+
+
+@schema
+class RecordingModality(dj.Lookup):
+     definition = """
+     recording_modality:   varchar(64)                  # modalities for recording (ephys, imaging, video_recording, etc.) 
+     ---
+     modality_description: varchar(255)                 # description for the modality
+     root_direcory:        varchar(255)                 # root directory where that modality is stored (e.g. ephys = /braininit/Data/eletrophysiology)
+     """
+     contents = [
+        ['ephys',             '', '/braininit/Data/eletrophysiology'],
+        ['imaging',           '', '/braininit/Data/eletrophysiology'],
+        ['video_acquisition', '', '/braininit/Data/video_acquisition']
+     ]
+
+
+@schema
+class PreprocessingParamSet(dj.Lookup):
+    definition = """
+    # Parameter set to be used in the preprocessing steps
+    preprocess_paramset_idx:  smallint
```

Comment:
Which steps do you consider pre-processing versus processing for the different modalities?  Thanks.

Reply:
I will consider preprocessing all the steps where the data is transformed before the sorting algorithm per se
For ephys, everything before kilosort, for imaging everything before suite2p

Reply:
Thanks!